### ISSUE TITLE
No HASH stats on dashboard. 

### ISSUE THREAD

**Issue by i8088** [bzminer/bzminer#23]
there is no HASH stats on dashboard. Please provide API host connection for Simplemining.   

Miner work but , no stats on Simplemining Dashboard.   0 h/s

**Comment by bzminer** [comment]
hey @i8088 , that was an issue with a previous release where the some pools disconnecting caused a reconnection loop where the connection wasn't able to be made again. latest version should fix that

**Comment by i8088** [comment]
Hello @bzminer it is not about version bug  .  I try last version now. Nothing fix  .    May be you should contact with simplemining  team .   Because of this issue I don't use your miner on my too many rigs.    

![bz](https://user-images.githubusercontent.com/33055151/150976982-859064cc-5219-4583-801e-c3e42c656b4e.JPG)

**Comment by paopao-tang** [comment]
Yes, I have the same problem. I can't see anything on the hive os dashboard, but mining is normal. Please solve it. Otherwise, you can't use your hoes.

**Comment by paopao-tang** [comment]
@bzminer

**Comment by i8088** [comment]
@bzminer @paopao-tang Developer is not sensitive about it. Go with T-rex miner

**Comment by paopao-tang** [comment]
@i8088  T-rex miner Does it support alpha?
